MHacks/RoundedView.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  RoundedView.swift
+//  MHacks
+//
+//  Created by Connor Krupp on 9/11/16.
+//  Copyright © 2016 MHacks. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable
+class RoundedView: UIView {
```

Comment:
Please don't use IBDesignable. I know it's cool but it makes storyboards unbearably slow on older computers. For this case you can really just add a line of code in the AnnouncementCell initializer that sets the corner radius. Also, the corner radius needs to be set to 1, not 2, since the width of the view is 2.